django/api/models/rebate_historical.py

```diff
@@ -0,0 +1,28 @@
+from shortuuid.django_fields import ShortUUIDField
+from django.conf import settings
+from django.db.models import (
+    CharField,
+    IntegerField,
+    DateField,
+    BooleanField,
+    PROTECT,
+    ForeignKey,
+)
+
+from django.core.validators import MinLengthValidator
+
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+from ..models import GoElectricRebateApplication
+
+
+class rebate_historical(TimeStampedModel):
+    id = ShortUUIDField(length=16, primary_key=True, editable=False)
+    application_id = ForeignKey(GoElectricRebateApplication, on_delete=PROTECT)

```

Comment:
Does this allow for null values?

Reply:
@naomiaro Unless specified with null=True,  null values will not be allowed by default. Should we allow null values for application_id field?

Reply:
Yes the historical applications won't have an application in our system, just new records we put in this table.